patterns/page-04-newsletter-landing.php

```diff
@@ -29,7 +30,8 @@
 
 <!-- wp:buttons {"layout":{"type":"flex","justifyContent":"center"}} -->
 <div class="wp-block-buttons"><!-- wp:button -->
-<div class="wp-block-button"><a class="wp-block-button__link wp-element-button"><?php echo esc_html_x( 'Sign up', 'sample content for newsletter subscription button', 'twentytwentyfour' ); ?></a></div>
+<div class="wp-block-button"><a class="wp-block-button__link wp-element-button">Sign up</a></div>
```

Comment:
Nit: Please make sure that all text is translation ready.

Reply:
I updated this.